src/Routing/QueryRouteEnhancer.php

```diff
@@ -17,7 +17,7 @@ class QueryRouteEnhancer implements EnhancerInterface {
    */
   public function enhance(array $defaults, Request $request) {
     $route = $defaults[RouteObjectInterface::ROUTE_OBJECT];
-    if (!$route->hasDefault('_graphql')) {
+    if (!$this->applies($route)) {
```

Comment:
I think I resolved that incorrectly.  I'll rebase.

Reply:
Please could you also fully enable PHP 7.4 on the CI, like [`6d767ee` (#1052)](https://github.com/drupal-graphql/graphql/pull/1052/commits/6d767ee54e2891402fad2b0bede9c303e966985a)? Thanks!

Reply:
We've used a patch based on this PR with just the `return` change below:

```php
return $defaults + ['operations' => $operations];
```

Would be great to get this PR fixed up and merged!

Thanks!